trunk/SUAVE/Input_Output/OpenVSP/vsp_propeller.py

```diff
@@ -111,7 +111,10 @@ def read_vsp_propeller(prop_id, units_type='SI',write_airfoil_file=True):
         prop.tag = tag
     else: 
         prop.tag = 'propgeom'
-
+    
```

Comment:
because the wing can in fact be scaled. It is typically one but if you import a geometry, lets say a turbofan nacelle into open VSP and want to scale it down to a drone, these lines do that. I will uncomment them